crates/services/txpool/src/test_helpers.rs

```diff
@@ -172,6 +178,16 @@ pub(crate) fn random_predicate(
     .into_default_estimated()
 }
 
+pub struct MockWasmChecker;
```

Comment:
nit: I'd name this `Fake` instead of `Mock`, since it's not "mocking" behavior.

Reply:
The terminology seems to be all over the place, e.g. https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing shows that there's no consensus around the naming. I don't think we them that consistently either.